lib/robots/dor_repo/base.rb

```diff
@@ -13,6 +13,17 @@ def seed_uri(druid)
         Dor::Services::Client.object(druid).find.label
       end
 
+      def original_site_uri(druid)
```

Comment:
Was there a need to put this functionality in `base.rb` rather than in `thumbnail_generator.rb`?

Reply:
I started here since capture_thumbnail used `seed_uri` before this change and it was in `base.rb`. `seed_uri` gets used in more than one place, however, so I guess that is why it was there. I see what you're saying so will try moving this out into a private method in `thumbnail_generator.rb`

Reply:
That helps me understand, there's no need to update it if you think it makes most sense this way.